snippets/python/s/is-armstrong.md

```diff
@@ -0,0 +1,38 @@
+---
+title: is-armstrong
+type: snippet
+language: python
+tags: [math]
+cover: balloons
+dateModified: 2023-09-12 22:30:04 +300
+---
+
+Checks whether the given number is an Armstrong Number.
+
+A positive integer of n digits is called an Armstrong number of order n (order is number of digits) if, 
+abcd... = a**n + b**n + c**n + d**n + .... 
+
+- Use a copy variable to make a copy of the given number.  
+- Initialize Sum=0. 
+- Find the total number of digits in the given number using len(str(copy)). 
+- Iterate through each digit using modulo(%) operator, raise it to the power of order and add the result to Sum.
+- Divide the number by 10 at each iteration and exit the loop when num becomes 0.
+- Return True if Sum is equal to copy of num, False if it is not equal to copy of num.
+
+```py
+def is_armstrong(num):
+    copy = num 
+    Sum = 0
```

Comment:
Not mandatory, but I think it is better if you follow the same naming convention all over the Python snippet. All the other variables in the snippet start with smaller case letters, without "Sum". So, it will be better if you rename this variable "sum".

Reply:
Hey, thanks for your feedback. But I used 'Sum' because 'sum' is an inbuilt function in python so it would be a bad practice even though it supports it. Maybe I should consider changing the variable name to total/result.